serverless.yml

```diff
@@ -0,0 +1,220 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: easy-n-dizzy 
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-1
```

Comment:
I see you intentionally changed region to `us-east-1`. This is not a mistake but I believe we should try doing that in scope of `eu-west-1` as much as possible. Please correct me if I'm wrong :)

Reply:
> I see you intentionally changed region to `us-east-1`. This is not a mistake but I believe we should try doing that in scope of `eu-west-1` as much as possible. Please correct me if I'm wrong :)

You're right, but deployment propagation to non-standard regions (like `eu-west-1`) can take up to 24 hours (see [this topic](https://aws.amazon.com/ru/premiumsupport/knowledge-center/s3-http-307-response/) for further details). I was able to deploy only on Saturday's afternoon, so in order to make the site being accessible for reviewers right from Saturday, 23:59:59, I decided to deploy it on standard (us-east-1) region.